internal/hns/hnsglobals.go

```diff
@@ -11,6 +11,8 @@ type HNSVersion struct {
 
 var (
 	HNSVersion1803 = HNSVersion{Major: 7, Minor: 2}
+	// default namespace ID used for all template and clone VMs.
+	DEFAULT_CLONE_NETWORK_NAMESPACE_ID = "89EB8A86-E253-41FD-9800-E6D88EB2E18A"
```

Comment:
Is this an arbitrary value you have chosen to be used for the namespace inside of cloned UVMs? If so, we should just define it internally where it is needed. Putting it here implies it is defined by HNS.

Reply:
Moved this inside the uvm folder.